app/Http/Resources/EntityResource.php

```diff
@@ -203,9 +203,23 @@ public function entity(array $prepared = [])
             $merged['entity_abilities'] = EntityAbilityResource::collection($misc->entity->abilities);
             $merged['entity_assets'] = EntityAssetResource::collection($misc->entity->assets);
         }
-
+        if ($misc->ancestors) {
```

Comment:
Let's only load these if related is set to true, to avoid it loading by default. Also can you make it work with APP_LAZY=true?

Reply:
Both $misc->ancestors and $misc->getAncestors() work with APP_LAZY set to true

Reply:
![image](https://github.com/owlchester/kanka/assets/3263440/8e900894-d825-40b4-8c38-eb1452b8d5e2)
I get an error about lazy on another relation directly when calling `locations?related=1` so I'm a bit confused how it's working on your end :)

Reply:
When I fix the missing with() for abilities, I now get the error for ancestors 
![image](https://github.com/owlchester/kanka/assets/3263440/b5097351-0c2a-45b5-be7e-378c994da704)